documentation/OtherMLPackages.md

```diff
@@ -0,0 +1,63 @@
+# Testing Other ML Packages
+
+In our documentation, we state that the mitigation algorithms in Fairlearn are compatible with any package which presents an interface similar to SciKit-Learn.
+Specifically, so long as there's some sort of 'estimator' concept which provides `fit()` and `predict()` methods (the former with a `sample_weight=` argument), then those estimators can be used with our mitigation algorithms.
+However, we do not currently have tests proving that we do offer this sort of compatibility with a variety of popular ML packages.
+
+## Target Packages
+
+We should show compatibility with the following ML packages:
+  - EBM
+  - XGBoost
+  - LightGBM
+  - TensorFlow
+  - PyTorch
```

Comment:
The sklearn wrapper for Pytorch has 2 issues.

1. The use of sample_weights [is not trivial.](https://skorch.readthedocs.io/en/stable/user/FAQ.html#i-want-to-use-sample-weight-how-can-i-do-this)

2. It works with DataFrame, but you need to use a sklearn.Pipeline which `fairlearn` doesn't support yet.

In addition, they are some type/shape errors that happen, I'll investigate.

I will continue to work on this. next week.

https://gist.github.com/Dref360/80c3b155276c41a879bf95ff72542133

Reply:
For sample_weight we'll soon have a workaround using resampling. I briefly mentioned this in a different comment.

For pipelines: what's needed to get compatible? This sounds worth doing :-)

Reply:
Thanks for looking at this - uncovering these issues is a major goal of this work.

Reply:
I submitted an issue for sklearn.Pipeline.
https://github.com/fairlearn/fairlearn/issues/580